src/app.js

```diff
@@ -123,14 +116,26 @@ class App {
         this.map.centerLocation(pos.coords);
         this.map.watchLocation();
         this.map.setMyLocation(pos.coords);
-      }, 
+      },
       () => this.showFallbackLocation());
   }
 
   showFallbackLocation() {
-    this.map.centerLocation(this.fallBackLocation,12);
+		let fallBackLocation = null;
+
+		switch (this.translation.currentLocale) {
+			case 'fr': {
+				fallBackLocation = defaultLocations.PARIS;
```

Comment:
I think you need to `break;`. When debugging I saw that the switch/case continues even if it goes into the `fr` branch.
Hence it doesn't work yet.

Reply:
Done.